### PR TITLE
Upgrade to Rust 2018

### PR DESCRIPTION
**Pull request by jstnlef**

### CODE REVIEW

File: src/packet/header/heart_beat_header.rs [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+use crate::error::NetworkResult;
+use crate::net::constants::HEART_BEAT_HEADER_SIZE;
+use crate::packet::PacketTypeId;
+use crate::protocol_version::ProtocolVersion;
 use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
-use error::NetworkResult;
-use net::constants::HEART_BEAT_HEADER_SIZE;
-use packet::PacketTypeId;
-use protocol_version::ProtocolVersion;
+
```
Comment (LucioFranco) [inline src/packet/header/heart_beat_header.rs]: can you remove this new line?

File: src/packet/header/standard_header.rs [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+use crate::net::constants::STANDARD_HEADER_SIZE;
+use crate::packet::PacketTypeId;
+use crate::protocol_version::ProtocolVersion;
 use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
-use error::NetworkResult;
-use infrastructure::DeliveryMethod;
-use net::constants::STANDARD_HEADER_SIZE;
-use packet::PacketTypeId;
-use protocol_version::ProtocolVersion;
+
```
Comment (LucioFranco) [inline src/packet/header/standard_header.rs]: and this one

Comment (jstnlef) [reply src/packet/header/standard_header.rs]: done 👍

File: src/packet/header/heart_beat_header.rs (hunk 1 above)
Comment (jstnlef) [reply src/packet/header/heart_beat_header.rs]: done 👍

**Comment by LucioFranco** [comment]
Rebased against master I've fixed CI, thanks.

**Comment by LucioFranco** [comment]
Can you squash these into one?

**Comment by jstnlef** [comment]
Done

**Comment by LucioFranco** [comment]
bors r=TimonPost,LucioFranco

**Pull request merged**